app/controllers/concerns/cms/generator_filter/rss.rb

```diff
@@ -2,40 +2,40 @@ module Cms::GeneratorFilter::Rss
   extend ActiveSupport::Concern
 
   private
-    def generate_node_rss(node, opts = {})
-      path = opts[:url] || "#{node.filename}/index.html"
-      return if Cms::Page.site(node.site).and_public.filename(path).first
+  def generate_node_rss(node, opts = {})
```

Comment:
__[rubocop]__
* Perceived complexity for generate_node_rss is too high. [15/14]

<!-- Comment by [SideCI](https://sideci.com) -->